### DISCUSSION TITLE
How to logout?

### DISCUSSION THREAD

**Discussion by Falci** [family/connectkit#156] [Q&A] [answered]
Is it possible to use only custom components?
I'm using my own button for login, but I'm, not sure how to logout avoiding the modal with user details.

**Comment by lochie** [comment, marked as answer]
you can use wagmi's [useDisconnect](https://wagmi.sh/react/hooks/useDisconnect) Hook to disconnect a connected wallet. If you're looking to sign a wallet out after using Sign In With Ethereum you can use the `signOut` function in our `useSIWE` Hook, which you can read about in our docs: https://docs.family.co/connectkit/usesiwe-hook#useSIWE-signOut
